Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.py

```diff
@@ -41,17 +42,30 @@ def sign_email(client: Client, args: Dict):
     """
```

Comment:
please add the `use_transport_encoding: bool` arg to `sign_email` method
and update the code by checking this arg once:

> if use_transport_encoding:
     your code part
  else:
      the old code
      
in the `main` method add:
`use_transport_encoding: bool = argToBoolean(demisto.params().get('use_transport_encoding'))`
and pass it to `sign_email` method

Reply:
use_transport_encoding is not a parameter at the moment, but an argument, as you might want to change this depending on the message. I set it in the beginning of the function and use the boolean for the check afterwards.

Reply:
@ckaadic you right its the command parameter, my bad.
what i meant in the last comment is to update the code by checking `use_transport_encoding` arg once inside `sign_email`, you check it 3 times, it should be:

> if use_transport_encoding:
> your code part
> else:
> the old code

the old part part should start from the line:
> message_body = args.get('message_body', '')
to:
> client.smime.write(out, p7, buf, SMIME.PKCS7_TEXT)